2021/day11/day11.java

```diff
@@ -0,0 +1,148 @@
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class day11 {
+    /*
+     * Creates a grid surrounded by a border of nines, e.g.:
+     * 999999
+     * 1234 912349
+     * 5678 --> 956789
+     * 9999 999999
+     * 999999
+     */
```

Comment:
It's hard to parse this example. Mention that it creates a 2D array with the energy levels surrounded by a border. Also, in my opinion this would work nicer with a `static ArrayList<ArrayList<Point>> neighbours(int x, int y, int width, int height` method instead of the border.